Leetcode/Array_Problem/Easy/1.Twosum.js

```diff
@@ -0,0 +1,57 @@
+/*Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to target.
+
+You may assume that each input would have exactly one solution, and you may not use the same element twice.
+
+You can return the answer in any order.
+
+Example 1:
+
+Input: nums = [2,7,11,15], target = 9
+Output: [0,1]
+Explanation: Because nums[0] + nums[1] == 9, we return [0, 1].
+Example 2:
+
+Input: nums = [3,2,4], target = 6
+Output: [1,2]
+Example 3:
+
+Input: nums = [3,3], target = 6
+Output: [0,1] 
+
+Input: nums = [0,4,3,0] target = 0
+Output: [0,3];*/
+
+// solution 1
+
+const getIndex = (arr, val) => arr.findIndex((ele) => ele === val);
```

Comment:
Write the comments here solution work if not testing the code leetcode test scenarios